app/locale/lv.coffee

```diff
@@ -3399,11 +3399,11 @@ module.exports = nativeDescription: "latviešu", englishDescription: "Latvian",
 #    failed: "Your payment failed, please try again"
 #    session_week_1: "1 session/week"
 #    session_week_2: "2 sessions/week"
-#    month_1: "Monthly"
-#    month_3: "Quarterly"
-#    month_6: "Half-yearly"
-#    year_1: "Yearly"
-#    most_popular: "Most Popular"
+#    month_1: "Ikmēneša"
```

Comment:
remove `#` from `payments:` above

Reply:
seems like you missed this @thehashicoder

Reply:
Indeed. Just fixed and added more translation. Thanks @mrfinch !